src/types/options/index.ts

```diff
@@ -152,20 +152,14 @@ export interface BlockOptions {
   options: Option[];
 }
 export interface MessageOptions {
-  options: {
-    text: string;
-    value: string;
-  }[];
+  options: Option[];
 }
 export interface DialogOptions {
-  options: {
```

Comment:
You need to revert this change. Dialog's option data structure here is correct. The legacy one is different from the latest Block Kit based one.

Reply:
Done, thank you for calling this out!